rollup/internal/controller/relayer/l2_relayer.go

```diff
@@ -423,107 +428,29 @@ func (r *Layer2Relayer) ProcessCommittedBatches() {
 	r.metrics.rollupL2RelayerProcessCommittedBatchesTotal.Inc()
 
 	batch := batches[0]
-	hash := batch.Hash
 	status := types.ProvingStatus(batch.ProvingStatus)
 	switch status {
 	case types.ProvingTaskUnassigned, types.ProvingTaskAssigned:
-		// The proof for this block is not ready yet.
-		return
-	case types.ProvingTaskVerified:
-		log.Info("Start to roll up zk proof", "hash", hash)
-		r.metrics.rollupL2RelayerProcessCommittedBatchesFinalizedTotal.Inc()
+		now := time.Now()
```

Comment:
need use `utils.NowUTC()`. because the time stored in db is utc time.

Reply:
changed in [9a6ac09](https://github.com/scroll-tech/scroll/pull/1004/commits/9a6ac09e79b292d16cc25f3316ed2cdaa5399a12)